### ISSUE TITLE
Dashboard: Update Firmware

### ISSUE THREAD

**Issue by syedkarim** [Outernet-Project/librarian#376]
Is it supposed to be empty (for now)?

**Comment by abgoyal** [comment]
Which build?

**Comment by syedkarim** [comment]
Definitely on 2.02a, but I've seen this problem on previous builds, as well.

**Comment by abgoyal** [comment]
I cannot replicate this. Could you give more info:

Build, browser, etc.

**Comment by syedkarim** [comment]
Are you able to see the upload form?

I was using Chrome on Ubuntu 14.04. This was the 0826 version of the build.

**Comment by abgoyal** [comment]
yes. i have the same version of the build on one of mine.

Oh. are you logged in?

**Comment by syedkarim** [comment]
Yes, definitely logged in (otherwise Dashboard is not visible). I'll check
again today--and take a screenshot.
ᐧ
